spec/integration/snapshot_query_examples_spec.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe 'Snapshot Query Examples' do
+  require_topology :replica_set, :sharded
+  require_stress
+  min_server_fcv '5.0'
+
+  let(:uri_string) do
+    "mongodb://#{SpecConfig.instance.addresses.join(',')}/?w=majority"
+  end
+
+  context "Snapshot Query Example 1" do
+    before do
+      client = Mongo::Client.new(uri_string, database: "pets")
+      client['cats'].delete_many({})
+      client['dogs'].delete_many({})
+
+      client['cats'].insert_one({
```

Comment:
While the curly braces `{}` in this case can be provided, they are redundant and most Ruby developers won't use them.  I suggest removing them.

Reply:
done